packages/wallet/backend/src/walletAddress/service.ts

```diff
@@ -159,62 +126,34 @@ export class WalletAddressService implements IWalletAddressService {
         publicName: args.publicName,
         accountId: args.accountId,
         id: rafikiWalletAddress.id,
-        isWM: args.isWM,
-        assetCode: webMonetizationAsset?.code,
-        assetScale: webMonetizationAsset?.scale
+        assetCode: null,
+        assetScale: this.env.MAX_ASSET_SCALE
       })
-
-      args.isWM &&
-        (await this.cache.set(walletAddress.id, walletAddress, {
-          expiry: 60
-        }))
     }
 
     return walletAddress
   }
 
-  async list(userId: string, accountId: string): Promise<WalletAddressList> {
+  async list(userId: string, accountId: string): Promise<WalletAddress[]> {
     const account = await this.accountService.findAccountById(accountId, userId)
 
     const walletAddressesResult = await WalletAddress.query()
       .where('accountId', account.id)
       .where('active', true)
 
-    const result = walletAddressesResult.reduce(
-      (acc, pp) => {
-        if (pp.isWM) {
-          acc.wmWalletAddresses.push(pp)
-        } else {
-          acc.walletAddresses.push(pp)
-        }
-        return acc
-      },
-      {
-        wmWalletAddresses: [] as WalletAddress[],
-        walletAddresses: [] as WalletAddress[]
-      }
-    )
-
-    return result
+    return walletAddressesResult
   }
 
   async listAll(userId: string): Promise<WalletAddress[]> {
     return WalletAddress.query()
-      .where({ isWM: false, active: true })
+      .where({ active: true })
       .joinRelated('account')
       .where({
         'account.userId': userId
       })
   }
 
   async getById(args: GetWalletAddressArgs): Promise<WalletAddress> {
-    //* Cache only contains WalletAddresses with isWM = true
-    const cacheHit = await this.cache.get(args.walletAddressId)
```

Comment:
as any WA address can be used for WM, should all WA be cached instead of removing it?

Reply:
I thought about this too, but I was curious if there's a reason why all addresses weren't already cached in the first place? If there isn't any, then I'm fine with caching all WA.

Reply:
the main reason why only WM WA where cached was because there are a lot of web monetization transactions in a short period of time. On simple WA there was no real reason to cache as the transactions will be dispersed in time.

As after these changes any WA can be used for WM it make sense to have the same caching mechanism for all WAs.